src/main/scala/models/JsonClass.scala

```diff
@@ -19,13 +19,13 @@ package models
 import play.api.libs.json.Json
 
 
-case class JsonClass(app:String,
+case class JsonClass(app: String,
```

Comment:
`JsonClass` is really non descriptive class name given that it looks like the class contains informations related to a log entry.